src/sentry/grouping/enhancer/__init__.py

```diff
@@ -463,6 +481,66 @@ def visit_quoted_ident(self, node, children):
         return node.match.groups()[0].lstrip("!")
 
 
+def _merge_values(frames: Sequence[dict[str, Any]], cache_key: str) -> None:
+    """This will merge the cached values if any is found for this stacktrace."""
+    changed_frames_values = cache.get(cache_key)
+    if not changed_frames_values:
+        return
+
+    try:
+        for frame, changed_frame_values in zip(frames, changed_frames_values):
+            frame["in_app"] = changed_frame_values["in_app"]
```

Comment:
Based on this:
https://github.com/getsentry/sentry/blob/9d06f817ed2d600a7e77378b33549222929c192c/src/sentry/grouping/enhancer/actions.py#L101-L105